int32 id
float64 latitude
float64 longitude
float32 speed
float32 heading
float32 acceleration
float32 brake
